### ISSUE TITLE
Call for Korean translators for document!

### ISSUE THREAD

**Issue by SooKim1110** [naver/fixture-monkey#864]
## For New Translators
The official documentation for Fixture Monkey can be found here:
- https://naver.github.io/fixture-monkey/docs/introduction/overview/

If this is your first time contributing to an open source library, please refer to the following guides: 
- https://naver.github.io/OpenSourceGuide/book/BetterContribution/why-contribute-to-open-source.html
- https://github.com/naver/fixture-monkey/blob/main/CONTRIBUTING.md

For communication related to Fixture Monkey, please use the Gitter channel: 
- https://app.gitter.im/#/room/#fixture-monkey_community:gitter.im.

## Korean Translation Guide
We are currently recruiting Korean translators for the Fixture Monkey official documentation (v1.0.x version)!

Translation Guide:

1. Look at the checklist below to find unassigned pages, and choose the page you want to translate (select 1 or 2 pages at a time).
2. Leave a comment below with the page you wish to translate.
3. Fork the repository and checkout to the `docs/kor-translation` branch, then create a new branch for your work.
4. The page to translate is located at the `docs -> content -> v1.0.x-kor` directory.
5. If you want to check your work locally, navigate to the docs directory(`cd docs`) in your console and run (`npm start`). 
![fixturemonkeydocs](https://github.com/naver/fixture-monkey/assets/47516074/9a9376fe-70e2-48dc-b17b-376a44155434)
6. Once your work is complete, submit a pull request from your branch to the `docs/kor-translation` branch.

## Content List
1. Introduction
  - [x] Overview - @graceful-martin 

2. Getting Startted
  - [x] Requirements - @songkg7 
  - [x] Creating test objects  - @songkg7 
  - [x] Creating test objects without Lombok - @songkg7
  - [x] Adding Bean Validation - @JoeCP17
  - [x] Customizing objects - @songkg7
  - [x] Creating objects in Kotlin - @songkg7

3. Generating Objects
  - [x] FixtureMonkey - @this-is-spear
  - [x] Generating Complex Types - @this-is-spear
  - [x] Instantiate Methods - @this-is-spear
  - [x] Introspector - @this-is-spear

4. Customizing Objects
  - [x] Fixture Customization APIs - @rachel5004
  - [x] Expressions - @rachel5004
  - [x] Customizing random values with Arbitrary - @doxxx93 
  - [x] InnerSpec - @songkg7 

7. Fixture Monkey Options
  - [x] Concepts - @Voyager003 
  - [x] Generation Options - @Voyager003 
  - [x] Customization Options - @Voyager003 
  - [x] Other Options - @Voyager003 

8. Plugins
- Kotlin Plugin
  - [x] Features - @hubtwork 
  - [x] Introspectors for Kotlin - @hubtwork 
  - [x] Kotlin DSL Exp - @hubtwork 

- Jackson Plugin
  - [x] Features - @graceful-martin 
  - [x] JacksonObjectArbitraryIntrospector  - @graceful-martin 
  - [x] Jackson Annotations  - @graceful-martin 

- Jakarta Validation Plugin
  - [x] Features - @jwp345
  - [x] Bean Validation - @jwp345 

- Kotest Plugin
  - [x] Features - @iamzin 
  - [x] Kotest Property Based Testing - @iamzin 

9. Intellij Plugin
  - [x] Fixture Monkey Helper - @vincentj2 

10. Cheat Sheet
  - [x] FAQ - @fingersdanny

-------------
In addition to Korean translators, we are also seeking contributions to the documentation! If you are interested, please check the discussion pages:

https://github.com/naver/fixture-monkey/discussions/858
https://github.com/naver/fixture-monkey/discussions/859

**Comment by 724thomas** [comment]
hi, can i pick this up?

**Comment by SooKim1110** [comment]
@724thomas Hi! Is there a specific page in the getting started section that you'd like to translate first? We recommend translating 1 to 2 pages at a time. You can translate more after submitting the PR! (I've assigned you to the overview page.)

**Comment by 724thomas** [comment]
> @724thomas Hi! Is there a specific page in the getting started section that you'd like to translate first? We recommend translating 1 to 2 pages at a time. You can translate more after submitting the PR! (I've assigned you to the overview page.)

아 한국분이시군요;
죄송합니다..

**Comment by songkg7** [comment]
I'd like to contribute as well.

I would appreciate it if you could assign me to the requirements section.

**Comment by this-is-spear** [comment]
@SooKim1110 Hello! 
I'd like to contribute to that fixture-monkey.

I would like to translate the `3.Generating Objects.FixtureMonkey` part.

**Comment by rachel5004** [comment]
Hi @SooKim1110 
I would like to work on this, can you please assign `4. Customizing Objects` to me?

**Comment by SooKim1110** [comment]
@songkg7, @this-is-spear, @rachel5004 Hello! Thanks for your interest. I've assigned you to the page.

@rachel5004. Please start with the `Fixture Customization APIs` page in the `4. Customizing objects section`. We suggest translating 1 to 2 pages at a time. This page is a bit long, so feel free to split it into several PRs! Happy translating! 🚀

**Comment by JoeCP17** [comment]
@SooKim1110 

hello. If possible, I would also like to contribute. `Add Bean Validation if possible ` Would it be okay if I contribute this part?

**Comment by SooKim1110** [comment]
@JoeCP17 Sure! Thanks for showing interest. I've assigned you to the page.

**Comment by rachel5004** [comment]
@SooKim1110 
I'd like to keep on [Expressions](https://naver.github.io/fixture-monkey/docs/customizing-objects/expressions/) page :)
Can you please assign to me?

**Comment by SooKim1110** [comment]
@rachel5004 Sure! I've assigned you to the page.

**Comment by Voyager003** [comment]
Hi, @SooKim1110 
I would like to work on this, can you please assign 7. [Fixture monkey options](https://naver.github.io/fixture-monkey/docs/fixture-monkey-options/concepts/) part?

**Comment by Voyager003** [comment]
Hi, @SooKim1110 
I hope to contribute fixture monkey as well. Can i contribute [Fixture Monkey Options](https://naver.github.io/fixture-monkey/docs/fixture-monkey-options/concepts/) part?

**Comment by Voyager003** [comment]
HI, @SooKim1110 
Can i continually work `Generation Options`, `Customization Options`, `Other Options` part in Fixture Monkey Options?

**Comment by tabadmin** [comment]
If you need yet, i want to work for it. Any translations needed?

**Comment by hubtwork** [comment]
If you needs, i want to participate in. Any translations needed?

**Comment by SooKim1110** [comment]
@Voyager003 Sure! Assigned you to it!
@hubtwork Thanks for showing interest. How about starting with the `Features` page in `Kotlin Plugin`? Assigned you to the page.

**Comment by hubtwork** [comment]
@SooKim1110 If you need, i want to translate all kotlin-plugins

**Comment by SooKim1110** [comment]
@hubtwork Thanks for your participation. Assigned you to the page!

**Comment by doxxx93** [comment]
HI, @SooKim1110
I'd be happy to handle the `Customizing random values with Arbitrary` section in Customizing Objects part.

**Comment by SooKim1110** [comment]
@doxxx93 Sure! Assigned you to it!

**Comment by iamzin** [comment]
Hi! @SooKim1110 
I'd like to work on `Kotest Plugin` part, so if you don't mind please assign me on that part. :)

**Comment by SooKim1110** [comment]
@iamzin Welcome to the community! Assigned you to the section.

**Comment by graceful-martin** [comment]
Hello. @SooKim1110 
I'm interested in Jackson Plugin part translation. Could you please assigned me that part?

**Comment by jwp345** [comment]
Hi! @SooKim1110
I'd like to work on Jakarta Validation Plugin part translation. Could you assigned me that part?

**Comment by SooKim1110** [comment]
@graceful-martin, @jwp345 Sure, assigned you to the page!

**Comment by SooKim1110** [comment]
@724thomas Hi! I wanted to check if you still have intentions to translate the overview page as the translation work is nearing completion. Feel free to let me know if you'd prefer someone else to handle it.

**Comment by fingersdanny** [comment]
Hi @SooKim1110 I'd like to work on translating ``10 Cheat Sheet. FAQ`` :)

**Comment by 724thomas** [comment]
> @724thomas Hi! I wanted to check if you still have intentions to translate the overview page as the translation work is nearing completion. Feel free to let me know if you'd prefer someone else to handle it.

네 지워주세요

**Comment by graceful-martin** [comment]
If you don't mind, Can I take the overview page translation?

**Comment by SooKim1110** [comment]
@fingersdanny, @graceful-martin Sure, assigned you to the page!

**Comment by vincentj2** [comment]
Hi @SooKim1110  I'd  like to work on translate `9. Intellij Plugin - Fixture Monkey Helper` Could you assigned me that part?

**Comment by SooKim1110** [comment]
@vincentj2 Hi! Assigned you to the page.
